topics/04_oop_basic/hw/multiple_inheritance.py

```diff
@@ -0,0 +1,32 @@
+class BaseClass:
```

Comment:
LGTM
Only suggestion (for future similar situation) add docstring at the beginning of file with hometask.